openedge.properties

```diff
@@ -18,3 +18,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.19.2.date=2023-01-10
 2.19.2.changelogUrl=https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.19.1
 2.19.2.downloadUrl=https://github.com/Riverside-Software/sonar-openedge/releases/download/V2.19.1/sonar-openedge-plugin-2.19.2.jar
+
+2.24.2.description=Version 2.24.2
+2.24.2.sqVersions=[8.9,LATEST]
```

Comment:
This range needs to start with the current LTS: `9.9`

Reply:
Changed